src/components/Empty/Header.js

```diff
@@ -72,16 +79,14 @@ export default class Header extends React.Component{
                     </div>
                 </div>
                 <div className="status2">
-                    <img src="img/search-line.svg"/>
                     <img src="img/notebook-line.svg" onClick={this.clickBook}/>
```

Comment:
Can you convert this to use the Icon component?

Reply:
Should be very simple. Very small change

Reply:
ok

Reply:
ready